test/performance/odp_sched_latency.c

```diff
@@ -569,7 +553,7 @@ static void usage(void)
 	       "Usage: ./odp_sched_latency [options]\n"
 	       "Optional OPTIONS:\n"
 	       "  -c, --count <number> CPU count, 0=all available, default=1\n"
-	       "  -d, --duration <number> Test duration in scheduling rounds (millions), default=%d, min=1\n"
+	       "  -d, --duration <number> Test duration in scheduling rounds (millions), default=10, min=1\n"
```

Comment:
When a default value is changed, it has to be changed in two places. I'm approving anyway, so you can decide what to do, but please re-think one more time.

Reply:
I'm leaving it like this since default values are not changed often, and it is easier to follow values in the code when not hidden behind defines.